### DISCUSSION TITLE
Recording of Selenium actions and export to PHPWebdriver format?

### DISCUSSION THREAD

**Discussion by RestOp** [php-webdriver/php-webdriver#981] [General]
Hello,

as i can see, all the extension are removed from browser stores.
There is a Selenium IDE Extension https://chrome.google.com/webstore/detail/selenium-ide/mooikfkahbdckldjjndioackbalphokd
But is has no way to export for as PHPWebdriver format
There is also Katalon Recorder but it also has not was to export as PHPWebdriver format.

So is there a revorder or converter to get code in PHPWebdriver format?

Thanks!

**Comment by zilles** [comment]
I would be interested in this as well.

I might try to write an extension to do it, but if it already exists I don't want to waste my time.
